apps/bmd/src/AppContestSingleSeat.test.tsx

```diff
@@ -81,4 +81,24 @@ it('Single Seat Contest', async () => {
   expect(getByText(candidate1).closest('button')!.dataset.selected).toBe(
     'false'
   )
+
+  // Deselect the first candidate
+  fireEvent.click(getByText(candidate0))
+
+  // Check that the aria label has deselected added
+  expect(
+    getByText(candidate0).closest('button')?.getAttribute('aria-label')
+  ).toContain('Deselected, ')
+  // Check that other candidates do not have deselected added
+  expect(
+    getByText(candidate1).closest('button')?.getAttribute('aria-label')
+  ).not.toContain('Deselected, ')
+
+  // After 1 second the deselected label is removed
```

Comment:
I lean towards not using comments that are verbatim what the code is doing. I fear they will become out of date with the code; say if a search and replace is done on the code.